src/components/Metrics.js

```diff
@@ -96,11 +136,11 @@ function Metric({ label, value, color }) {
   )
 }
 
-function TokenBalance({ label, token, value }) {
+function TokenBalance({ label, token, value, currency }) {
   const theme = useTheme()
 
```

Comment:
here we could do `const {currencyRate, currencySymbol} = currency` and use those ones